tests/Functional/Visitor/Php/Symfony/ValidationAnnotationTest.php

```diff
@@ -41,11 +40,10 @@ public function testExtractAnnotation()
 
     public function testExtractAnnotationError()
     {
-        $factory = new LazyLoadingMetadataFactory(new AnnotationLoader(new AnnotationReader()));
+        $factory = new LazyLoadingMetadataFactory(new AttributeLoader());
         $extractor = new ValidationAnnotation($factory);
         $collection = $this->getSourceLocations($extractor, Resources\Php\Symfony\ValidatorAnnotationError::class);
 
-        $errors = $collection->getErrors();
-        $this->assertCount(1, $errors);
+        $this->assertCount(0, $collection);
```

Comment:
This looks like we just supress the test, i.e. the errors never tested?

Reply:
Well thats actually the point we discussed [here](https://github.com/php-translation/extractor/pull/175#discussion_r1626616867).

The `Symfony\Component\Validator\Mapping\Loader\AttributeLoader;` apparently does not give any errors when it encounters an an attribute it does not recognize, while apparently the `Symfony\Component\Validator\Mapping\Loader\AnnotationLoader` did when it encountered an unknown annotation.

Reply:
Ah I remember, thanks!

yeah, probably we don't need that test at all then, IMO it has no value anymore

Reply:
I agree indeed, does not really add much. I can remove it, but I am not able to do so at this moment, so that would have to wait until later this day.